### PR TITLE
Migrate to latest ODB changes regarding sequence offsets

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: explore/src/clue/scala/queries/common/ObsQueriesGQL.scala [hunk 1]
```diff
@@ -28,21 +28,10 @@ object ObsQueriesGQL:
   @GraphQL
   trait SequenceOffsets extends GraphQLOperation[ObservationDB]:
     val document = s"""
-      fragment stepDataGN on GmosNorthStep {
+      fragment stepData on GmosNorthStep {
```
Comment (toddburnside) [inline explore/src/clue/scala/queries/common/ObsQueriesGQL.scala:31]: This works for GmosSouth, too, even though it specifies it is for GmosNorthStep?

**Review by toddburnside — APPROVED** [review]
LGTM

File: explore/src/clue/scala/queries/common/ObsQueriesGQL.scala (hunk 1 above)
Comment (rpiaggio) [reply explore/src/clue/scala/queries/common/ObsQueriesGQL.scala:31]: Good catch. It does seem to work. But I'll revert, just in case.

Comment (toddburnside) [reply explore/src/clue/scala/queries/common/ObsQueriesGQL.scala:31]: Maybe grackle is not enforcing that.

**Review by toddburnside — APPROVED** [review]
LGTM

**Pull request merged**